src/smtencoding/FStar.SMTEncoding.Encode.fs

```diff
@@ -1509,188 +1576,265 @@ let encode_top_level_vals env bindings quals =
 let encode_top_level_let :
     env_t -> (bool * list<letbinding>) -> list<qualifier> -> list<decl> * env_t =
     fun env (is_rec, bindings) quals ->
+
     let eta_expand binders formals body t =
-        let nbinders = List.length binders in
-        let formals, extra_formals = BU.first_N nbinders formals in
-        let subst = List.map2 (fun (formal, _) (binder, _) -> NT(formal, S.bv_to_name binder)) formals binders in
-        let extra_formals = extra_formals |> List.map (fun (x, i) -> {x with sort=SS.subst subst x.sort}, i) |> U.name_binders in
-        let body = Syntax.extend_app_n (SS.compress body) (snd <| U.args_of_binders extra_formals) (Some <| (SS.subst subst t).n) body.pos in
-        binders@extra_formals, body in
+      let nbinders = List.length binders in
+      let formals, extra_formals = BU.first_N nbinders formals in
+      let subst = List.map2 (fun (formal, _) (binder, _) -> NT(formal, S.bv_to_name binder)) formals binders in
+      let extra_formals = extra_formals |> List.map (fun (x, i) -> {x with sort=SS.subst subst x.sort}, i) |> U.name_binders in
+      let body = Syntax.extend_app_n (SS.compress body) (snd <| U.args_of_binders extra_formals) (Some <| (SS.subst subst t).n) body.pos in
+      binders@extra_formals, body
+    in
```

Comment:
Don't mean to grumble too much ... but so many of these sorts of formatting change makes this diff very hard to review.

My understanding was that until such time as we can enforce a consistent style convention automatically (e.g., using the new pretty printer) edits to tweak whitespace conventions are not encouraged (since this will not necessarily converge)